src/tests/TestSettingsJSONSerialization/generate_test_case.py

```diff
@@ -136,6 +136,8 @@ def getDefaultValueForType(dataType):
         return "Settings::NoiseCancelBoth"
     elif dataType in ["EchoCancelOptionID"]:
         return "EchoCancelOptionID::SPEEX_MULTICHANNEL"
+    elif dataType in ["ConnectedQuitBehaviorID"]:
+        return "ConnectedQuitBehaviorID::ASK"
```

Comment:
You can't use the same default that you are also using in the settings here.

Reply:
What is the default value then? Or should I just use some other value but not the default declared within the settings.h?